README.md

```diff
@@ -652,7 +652,19 @@ nt-dates/data/](https://amiibereval2018.wordpress.com/important-dates/data/)
 * Platform: Twitter 
 * Medium: Text 
 * Reference: Ousidhoum, N., Lin, Z., Zhang, H., Song, Y. and Yeung, D., 2019. Multilingual and Multi-Aspect Hate Speech Analysis. ArXiv,. 
-  
+
+#### The Hateful Memes Challenge: Detecting Hate Speech in Multimodal Memes
+* Link to publication: [https://arxiv.org/pdf/2005.04790.pdf](https://arxiv.org/pdf/2005.04790.pdf)
+* Link to data: [https://www.drivendata.org/competitions/64/hateful-memes/](https://www.drivendata.org/competitions/64/hateful-memes/)
```

Comment:
this should link to the dataset direct download

Reply:
Ok, I will submit a new one~